cargo-dist/src/build/cargo.rs

```diff
@@ -223,6 +224,16 @@ pub fn build_cargo_target(
     // Process all the resulting binaries
     expected.process_bins(dist_graph, manifest)?;
 
+    let system_id = format!("build:local:{}", target.target_triple);
```

Comment:
It feels like something has gone wrong if we're synthesizing this id here -- isn't the current id in the WorkspaceGraph?

Reply:
Oh is this grappling with the fact that the "system" keys are too coarse (concatenating target triples when you try to build multiple on one machine?)